test/ERC1271.t.sol

```diff
@@ -81,7 +81,7 @@ contract ERC1271Test is DelegationHandler {
     function test_isValidSignature_invalidSignatureLength_reverts() public {
         bytes32 hash = keccak256("test");
         bytes memory signature = new bytes(63);
-        vm.expectRevert("Not implemented");
+        vm.expectRevert();
```

Comment:
is there a custom error from the ECDSA lib?>

Reply:
no cause its reverting bc of the abi.decode call, since we can process signatures whos length is not the exactly 64 or 65